proposed/cache.md

```diff
@@ -348,6 +359,17 @@ interface CacheItemPoolInterface
     public function save(CacheItemInterface $item);
 
     /**
+     * Persists multiple cache items immediately.
+     *
+     * @param array|\Traversable $items
+     *   A traversable collection or array of CacheItemInterface[] items
```

Comment:
Shouldn't this be `array of CacheItemInterface items`? Each is a cache item and not a collection of cache items.